lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -369,6 +426,18 @@ class EdgeBase {
     }
   }
 
+  /**
+   *
+   * @param {number} x1
+   * @param {number} y1
+   * @param {number} x2
+   * @param {number} y2
+   * @param {number} x3
+   * @param {number} y3
+   * @returns {number}
+   * @private
+   * @static
```

Comment:
I have no problem removing it, but when you say `Technically not static`, how so? because the static keyword is not used?